cdap-client/src/main/java/co/cask/cdap/client/MetadataClient.java

```diff
@@ -119,25 +121,25 @@ public MetadataClient(ClientConfig config) {
 
   /**
    * @param id the entity for which to retrieve metadata properties
+   * @param scope the {@link MetadataScope} to retrieve the properties from. If unspecified, this method retrieves
+   *              properties from both {@link MetadataScope#SYSTEM} and {@link MetadataScope#USER}
    * @return The metadata properties for the entity.
    */
-  public Map<String, String> getProperties(Id id)
```

Comment:
Worth it to keep a version of this method without a second parameter, as opposed to having the user pass in null?